### ISSUE TITLE
Добавить возможность импорта csv cо списком Товаров.

### ISSUE THREAD

**Issue by Nebula100** [YoshiOne/fingen#16]
Не плохо было бы сделать возможность импорта csv формата FinGen со списком товаров для каждой транзакции. Сейчас импортируются только транзакции.
То есть если сделать экспорт данных с опцией "Выгружать товары" а потом импортировать обратно -  получится куча транзакций без списка товаров с дублями по времени.

**Comment by YoshiOne** [comment]
Да, импорт/экспорт товаров есть в планах, пока реализован только экспорт.
